MBBSEmu/DOS/ExeRuntime.cs

```diff
@@ -145,7 +145,10 @@ private void SetupEnvironmentVariables()
                 bytesWritten += (ushort)(str.Length);
             }
             // null terminate
-            Memory.SetWord(ENVIRONMENT_SEGMENT, bytesWritten, 0);
+            Memory.SetByte(ENVIRONMENT_SEGMENT, bytesWritten++, 0);
+
```

Comment:
didn't that document you linked to me have a DWORD value of 0x0001 sitting between the end of environment and the EXE string?

Reply:
Is that the 2 `inc si` calls?